src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentPathTwigExtension.php

```diff
@@ -77,9 +77,24 @@ public function getContentPath($url, $webspaceKey = null, $locale = null)
                 $locale ?: $this->requestAnalyzer->getCurrentLocalization()->getLocalization(),
                 $webspaceKey
             );
-
             if (sizeof($portalUrls) > 0) {
-                return rtrim($portalUrls[0], '/');
+                if ($domain) {
```

Comment:
Not sure if the twig-extension is the right place to validate the sitemap urls. This extension is heavily used in other places. @danrot WDYT?